.gitignore

```diff
@@ -1,2 +1,7 @@
 node_modules
-lib
+index.js
+index.d.ts
+PortalManager.js
+PortalManager.d.ts
+server.js
+server.d.ts
```

Comment:
I don't understand the reasoning behind moving files to the root of the project. It's much easier to set it to lib and forget it than making sure that this list of files is up-to-date across .gitignore and package.json

Reply:
Sorry, this is the result of poor typescript-fu on my behalf. Can I ask that you make the necessary changes here? I Just couldn't work it out :/